www/casestudies/items/endtoend/index.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+include_once '../../../config.php';
+include_once 'includes/header.php';
+?>
+
+<div class="jumbotron section">
+    <div class="pull-right toolbar">
+        <ul class="list-inline">
+            <li data-toggle="tooltip" data-original-title="Toggle product overview box"><a href="#"><span class="glyphicon glyphicon-chevron-up jumbotron-toggle"></span></a></li>
+        </ul>
+    </div>
+    <h1>End to End Demo</h1>
+    <div class="section-intro">
+        <p>This is an End to End demo covering the main areas of Learnosity APIs.<p>
+        <p>Starting <strong>Authoring</strong> questions to turn them into an <strong>Assessment</strong> and finally <strong>Reporting</strong> to review the results and provide feedback.</p>
```

Comment:
<p>Starting with <strong>Authoring</strong> questions to turn them into an <strong>Assessment</strong> and finally <strong>Reporting</strong> to review the results and provide feedback.</p>